pagopa/io-app
# URL fix

## Short description
This pr fixes some deprecated urls

## List of changes proposed in this pull request
- [.env, ts/config.ts]: added two new urls.
- [ts/navigation/AuthenticationNavigator.tsx, ts/navigation/params/AuthenticationParamsList.ts, ts/navigation/routes.ts, ts/screens/authentication/SpidCIEInformationScreen.tsx, ts/screens/authentication/SpidInformationScreen.tsx]: removed unused screens.
- ts/screens/authentication/LandingScreen.tsx: replaced old screens with new url.
- ts/screens/authentication/cie/CiePinScreen.tsx: replaced deprecated url.

## How to test
On landing screen, try to click on `nospid-nocie button`: it should open the defined url.
On Cie access, try to click on `bottomSheetCTA` (Did you forget your PIN? )button:  it should open the defined url.
